src/modules/content-tree/components/list-item/list.item.component.js

```diff
@@ -71,9 +71,12 @@ class ListItem extends Component {
 
         return (
             <li {...itemAttrs}>
-                <a className="c-list-item__label">
-                    <span {...togglerAttrs} /> {this.renderIcon()} {name}
-                </a>
+                <div className="c-list-item__label">
+                    <span {...togglerAttrs} />{' '}
```

Comment:
Maybe it would be better to remove `{' '}` and add some additional CSS styling instead?

Reply:
That is strange, because I don't remember I added it. I'll remove it.